hw/ip/spi_host/data/spi_host.hjson

```diff
@@ -20,11 +20,11 @@
   sw_checklist:       "/sw/device/lib/dif/dif_spi_host",
   revisions: [
     {
-      version:            "2.1.0",
+      version:            "3.0.0",
       life_stage:         "L1",
-      design_stage:       "D2S",
-      verification_stage: "V2S",
-      dif_stage:          "S2",
+      design_stage:       "D0",
+      verification_stage: "V0",
+      dif_stage:          "S1",
```

Comment:
@a-will @Razer6: Does this look right to you (please see the message of the first commit for the reasoning behind these values)?

Reply:
Sadly, yes, that makes sense. I suppose if we end up making only small changes for a release's major revision bump, then we could potentially jump straight to D3/V3 sign-off reviews (so we don't have a long sequence of meetings for every release).